server/handle.go

```diff
@@ -127,6 +127,9 @@ func (l *listener4) HandleMsg4(buf []byte, oob *ipv4.ControlMessage, _peer net.A
 		tmp.UpdateOption(dhcpv4.OptMessageType(dhcpv4.MessageTypeOffer))
 	case dhcpv4.MessageTypeRequest:
 		tmp.UpdateOption(dhcpv4.OptMessageType(dhcpv4.MessageTypeAck))
+	case dhcpv4.MessageTypeInform:
+		tmp.UpdateOption(dhcpv4.OptMessageType(dhcpv4.MessageTypeAck))
+		tmp.YourIPAddr = nil
```

Comment:
This seems superfluous, are you sure this is needed? It should already be nil at this point, and in any case other plugins that might override it run later. 
```suggestion
```